pkg/installer/uninstall.go

```diff
@@ -231,6 +248,32 @@ func (in *Installer) uninstallEtcdOperator() error {
 	return err
 }
 
+// checkForExistingWorkloads ensures no bound PVCs are using a storageos storageclass.
+func (in *Installer) checkForExistingWorkloads() error {
+	pvcList, err := pluginutils.ListPersistentVolumeClaims(in.clientConfig, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	scList, err := pluginutils.ListStorageClasses(in.clientConfig, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, pvc := range pvcList.Items {
+		if pvc.Status.Phase != corev1.ClaimBound {
+			continue
+		}
+		if provisioner, ok := pvc.GetAnnotations()["volume.beta.kubernetes.io/storage-provisioner"]; ok && provisioner == stosSCProvisioner {
+			return fmt.Errorf(errWorkloadsExist, pvc.Name, *pvc.Spec.StorageClassName)
```

Comment:
Check for nil before dereferencing `*pvc.Spec.StorageClassName` or it will panic.  StorageClassName will be nil if the default StorageClass is used.